plume-common/src/utils.rs

```diff
@@ -23,8 +22,13 @@ pub fn make_actor_id(name: &str) -> String {
         .collect()
 }
 
+/**
+* Redirects to the login page with a given message.
+*
+* Note that the message should be translated before passed to this function.
+*/
 pub fn requires_login<T: Into<Uri<'static>>>(message: &str, url: T) -> Flash<Redirect> {
-    Flash::new(Redirect::to(format!("/login?m={}", gettext(message.to_string()))), "callback", url.into().to_string())
+    Flash::new(Redirect::to(format!("/login?m={}", message)), "callback", url.into().to_string())
```

Comment:
You are right, I forgot to call it from the routes (I preferred to change this behavior to avoid having to depend on rocket_i18n in plume-common, and to have an extra `Catalog` parameter).

Reply:
I totally understand your thought, it will be more logical to do so